src/dreammaker/error.rs

```diff
@@ -73,12 +76,14 @@ impl Context {
 
     /// Push an error or other diagnostic to the context.
     pub fn register_error(&self, error: DMError) {
-        if let Some(severity) = self.print_severity {
-            if error.severity <= severity {
-                let stderr = termcolor::StandardStream::stderr(termcolor::ColorChoice::Auto);
-                self.pretty_print_error(&mut stderr.lock(), &error)
-                    .expect("error writing to stderr");
-            }
+        if self.config.printable_error(&error) {
+            let stderr = termcolor::StandardStream::stderr(termcolor::ColorChoice::Auto);
+            self.pretty_print_error(&mut stderr.lock(), &error)
+                .expect("error writing to stderr");
+        }
+        // ignore errors with severity above configured level
+        if !self.config.registerable_error(&error) {
```

Comment:
I don't think I'm actually seeing the ability to change the severity of a diagnostic